models/tasks.js

```diff
@@ -281,6 +283,14 @@ const fetchActiveTaskMembers = async () => {
 const fetchTask = async (taskId) => {
   try {
     const task = await tasksModel.doc(taskId).get();
+    const extensionRequestSnapshot = await extensionRequestsModel.where("taskId", "==", taskId).get();
+    const extensionRequestsData = extensionRequestSnapshot.docs.map((doc) => {
+      const extensionRequestId = doc.id;
+      return { extensionRequestId, status: doc.get("status") };
```

Comment:
what if "status" is not available..?

Reply:
@iamitprakash 
```const extensionRequestSnapshot = await extensionRequestsModel
      .where("taskId", "==", taskId)
      .where("status", "==", EXTENSION_REQUEST_STATUS.PENDING)
      .count()
      .get();
    const extensionRequestsData = extensionRequestSnapshot.data().count;```
    
    after the suggestion of @Ajeyakrishna-k have updated the logic for getting the extensionRequest. I guess that won;t cause issue now?